Onion.Infrastructure/Services/LoggerService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Onion.Core.Interfaces.Services;
+using Onion.Core.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+
+namespace Onion.Infrastructure.Services
+{
+    public  class LoggerService : ILoggerService
+    {
+        public Error GetError(string id)
+        {
+            //2020-10-06
+
+            var xml = "<log>" + System.IO.File.ReadAllText(@$"C:\Users\Victor Ayala\source\repos\OnionArchitecture\OnionArchitecture\Onion\bin\Debug\netcoreapp3.1\Logs\{DateTime.Now.ToString("yyyy-MM-dd")}.xml") + "</log>";
+
+            Error model = new Error();
```

Comment:
I think the coding standards we have indicate to use var when it's apparent what the type will be. In this case you could go with var model = new Error(). Yet model is likely not a good name for this. How about var modelError = new Error();

Reply:
Working on it